rdr_service/genomic/genomic_job_components.py

```diff
@@ -2239,11 +2241,79 @@ def create_c1_genomic_participants(self, from_date, local=False):
             logging.info(f'Cohort 1 Participant Workflow: No participants to process.')
             return GenomicSubProcessResult.NO_FILES
 
+    def create_long_read_genomic_participants(self):
+        """
+        Create long_read participants that are already in the genomic system,
+        based on downstream filters.
+        :return:
+        """
+        participants = self._get_long_read_participants(max_num=384)
+
+        if len(participants) > 0:
+            return self.process_genomic_members_into_manifest(
+                participants=participants
+            )
+
+        logging.info(f'Long Read Participant Workflow: No participants to process.')
+        return GenomicSubProcessResult.NO_FILES
+
+    def process_genomic_members_into_manifest(self, *, participants):
+        """
+        Compiles AW0 Manifest from already submitted genomic members.
+        :param participants:
+        :return:
+        """
+
+        new_genomic_set = self._create_new_genomic_set()
+        processed_members = []
+        count = 0
+        # duplicate genomic set members
+        with self.member_dao.session() as session:
+            for i, participant in enumerate(participants):
+                dup_member_obj = GenomicSetMember(
+                    biobankId=participant.biobankId,
+                    genomicSetId=new_genomic_set.id,
+                    participantId=participant.participantId,
+                    nyFlag=participant.nyFlag,
+                    sexAtBirth=participant.sexAtBirth,
+                    collectionTubeId=participant.collectionTubeId,
+                    validationStatus=participant.validationStatus,
+                    validationFlags=participant.validationFlags,
+                    ai_an=participant.ai_an,
+                    genomeType='long_read',
```

Comment:
Can you put this in a variable? Similar to these:
` _ARRAY_GENOME_TYPE = "aou_array"`
` _WGS_GENOME_TYPE = "aou_wgs"`

Reply:
Sure.
https://github.com/all-of-us/raw-data-repository/pull/2270/commits/32a1ed82658f62b9b1b3d02acd909f652dd2484d